### ISSUE TITLE
/bin/bash: dev/vitunes: No such file or directory (on 10.7.3 w/iTunes 10.6.1)

### ISSUE THREAD

**Issue by zettabyte** [danchoi/vitunes#10]
With vitunes installed on my mac running Lion (10.7.3) with iTunes 10.6.1 the plugin always fails with the following message:

```
/bin/bash: dev/vitunes: No such file or directory
```

To reproduce I start iTunes and begin playing an album. Then, from the terminal I run `vitunes` to launch vim. I'm then given the ViTunesBuffer which contains:

```
Welcome to ViTunes

Press ,? for help
```

I then try pressing `.` in order to have vitunes show the current track. At this point, the following is displayed at the bottom of the vim screen beneath the status line:

```
/bin/bash: dev/vitunes: No such file or directory
 | /bin/bash: dev/vitunes: No such
Press ENTER or type command to continue
```

Attempting to run pretty much any other command ultimately has identical or equivalent results.

**Comment by danchoi** [comment]
If you type `gem list vitunes` what version of vitunes do you see installed?

**Comment by zettabyte** [comment]
I've got `0.4.8`:

```
$ gem list vitunes | grep vitunes
vitunes (0.4.8)
```

**Comment by zettabyte** [comment]
Hmm. It appears that for some reason the plugin wasn't getting loaded.

I use the `janus` "Vim Distribution" (https://github.com/carlhuda/janus). Consequently, I moved the generated `vitunes.vim` (based on the `plubin.erb` template) to a `~/.janus/vitunes/plugin` directory (which gets loaded by `pathogen`).

I'm not sure why, but it wasn't getting loaded but now it is. So it now works for me -- closing the issue.

However, I still think that running the ruby `vitunes` executable as I did above, which sources the plugin directly and executes the `ViTunes()` function, ought to work regardless. However, that's a separate issue.
